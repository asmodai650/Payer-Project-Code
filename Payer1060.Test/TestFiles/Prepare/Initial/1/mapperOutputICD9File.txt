CLAIM_ID|ICD9_CODE|PROJECT_ID|ICD9_TYPE|ADMITTING_CODE|PRINCIPAL_CODE|ORDER_IN_CLAIM|CLAIM_LINE_ID|FEED_ID|DATE_CREATED|DATE_UPDATED|USERNAME|CLAIM_NO|LINE_NO
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.881||NovusLoading|23261510|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.881||NovusLoading|23261510|, 23261510|C86.4
23261510|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.893||NovusLoading|23263555|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.893||NovusLoading|23263555|, 23263555|C86.4
23263555|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23263799|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23263799|, 23263799|C86.4
23263799|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23264780|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23264780|, 23264780|C86.4
23264780|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23264813|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23264813|, 23264813|C86.4
23264813|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23269549|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23269549|, 23269549|C86.4
23269549|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23270345|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.894||NovusLoading|23270345|, 23270345|C86.4
23270345|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.895||NovusLoading|23270514|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.895||NovusLoading|23270514|, 23270514|C86.4
23270514|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.895||NovusLoading|23270519|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.895||NovusLoading|23270519|, 23270519|C86.4
23270519|5A09357)
(|C86.4|1454|DIAG10|0|1|0||1|2022-06-17 15:10:32.895||NovusLoading|23270520|
|5A09357|1454|PROC10|0|1|0||1|2022-06-17 15:10:32.895||NovusLoading|23270520|, 23270520|C86.4
23270520|5A09357)
