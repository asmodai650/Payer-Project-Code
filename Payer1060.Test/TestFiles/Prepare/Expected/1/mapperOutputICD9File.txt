CLAIM_ID|ICD9_CODE|PROJECT_ID|ICD9_TYPE|ADMITTING_CODE|PRINCIPAL_CODE|ORDER_IN_CLAIM|CLAIM_LINE_ID|FEED_ID|DATE_CREATED|DATE_UPDATED|USERNAME|CLAIM_NO|LINE_NO
|S72.401A|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23261510|
|M97.11XA|1454|DIAG10|1|0|1||1|2022-07-21||NovusLoading|12-23261510|
|B37.0|1454|DIAG10|0|0|2||1|2022-07-21||NovusLoading|12-23261510|
|C92.01|1454|DIAG10|0|0|3||1|2022-07-21||NovusLoading|12-23261510|
|D68.69|1454|DIAG10|0|0|4||1|2022-07-21||NovusLoading|12-23261510|
|D84.821|1454|DIAG10|0|0|5||1|2022-07-21||NovusLoading|12-23261510|
|E27.1|1454|DIAG10|0|0|6||1|2022-07-21||NovusLoading|12-23261510|
|I69.951|1454|DIAG10|0|0|7||1|2022-07-21||NovusLoading|12-23261510|
|M54.9|1454|DIAG10|0|0|8||1|2022-07-21||NovusLoading|12-23261510|
|T38.0X5A|1454|DIAG10|0|0|9||1|2022-07-21||NovusLoading|12-23261510|
|E03.9|1454|DIAG10|0|0|10||1|2022-07-21||NovusLoading|12-23261510|
|E78.00|1454|DIAG10|0|0|11||1|2022-07-21||NovusLoading|12-23261510|
|E78.5|1454|DIAG10|0|0|12||1|2022-07-21||NovusLoading|12-23261510|
|F32.5|1454|DIAG10|0|0|13||1|2022-07-21||NovusLoading|12-23261510|
|F40.240|1454|DIAG10|0|0|14||1|2022-07-21||NovusLoading|12-23261510|
|G40.909|1454|DIAG10|0|0|15||1|2022-07-21||NovusLoading|12-23261510|
|G90.9|1454|DIAG10|0|0|16||1|2022-07-21||NovusLoading|12-23261510|
|I10|1454|DIAG10|0|0|17||1|2022-07-21||NovusLoading|12-23261510|
|I48.0|1454|DIAG10|0|0|18||1|2022-07-21||NovusLoading|12-23261510|
|I70.0|1454|DIAG10|0|0|19||1|2022-07-21||NovusLoading|12-23261510|
|I73.9|1454|DIAG10|0|0|20||1|2022-07-21||NovusLoading|12-23261510|
|K21.9|1454|DIAG10|0|0|21||1|2022-07-21||NovusLoading|12-23261510|
|K44.9|1454|DIAG10|0|0|22||1|2022-07-21||NovusLoading|12-23261510|
|Z88.5|1454|DIAG10|0|0|23||1|2022-07-21||NovusLoading|12-23261510|
|0SPC0JZ|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23261510|
|0SRC0J9|1454|PROC10|0|0|1||1|2022-07-21||NovusLoading|12-23261510|
|0SBC0ZZ|1454|PROC10|0|0|2||1|2022-07-21||NovusLoading|12-23261510|
|3E0U029|1454|PROC10|0|0|3||1|2022-07-21||NovusLoading|12-23261510|
|Q05.2|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23263555|
|5A1955Z|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23263555|
|5A12012|1454|PROC10|0|0|1||1|2022-07-21||NovusLoading|12-23263555|
|0UQMXZZ|1454|PROC10|0|0|2||1|2022-07-21||NovusLoading|12-23263555|
|3E033XZ|1454|PROC10|0|0|3||1|2022-07-21||NovusLoading|12-23263555|
|S43.402A|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23263799|
|8E0ZXY6|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23263799|
|U07.1|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23264780|
|R00.0|1454|DIAG10|0|0|1||1|2022-07-21||NovusLoading|12-23264780|
|R07.89|1454|DIAG10|0|0|2||1|2022-07-21||NovusLoading|12-23264780|
|I10|1454|DIAG10|0|0|3||1|2022-07-21||NovusLoading|12-23264780|
|E11.65|1454|DIAG10|0|0|4||1|2022-07-21||NovusLoading|12-23264780|
|D72.829|1454|DIAG10|0|0|5||1|2022-07-21||NovusLoading|12-23264780|
|Z79.4|1454|DIAG10|0|0|6||1|2022-07-21||NovusLoading|12-23264780|
|0T778DZ|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23264780|
|I21.4|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23264813|
|Z68.41|1454|DIAG10|0|0|1||1|2022-07-21||NovusLoading|12-23264813|
|I25.110|1454|DIAG10|0|0|2||1|2022-07-21||NovusLoading|12-23264813|
|E66.01|1454|DIAG10|0|0|3||1|2022-07-21||NovusLoading|12-23264813|
|I10|1454|DIAG10|0|0|4||1|2022-07-21||NovusLoading|12-23264813|
|Z79.82|1454|DIAG10|0|0|5||1|2022-07-21||NovusLoading|12-23264813|
|Z79.84|1454|DIAG10|0|0|6||1|2022-07-21||NovusLoading|12-23264813|
|E78.5|1454|DIAG10|0|0|7||1|2022-07-21||NovusLoading|12-23264813|
|E11.51|1454|DIAG10|0|0|8||1|2022-07-21||NovusLoading|12-23264813|
|D69.6|1454|DIAG10|0|0|9||1|2022-07-21||NovusLoading|12-23264813|
|E11.65|1454|DIAG10|0|0|10||1|2022-07-21||NovusLoading|12-23264813|
|E78.00|1454|DIAG10|0|0|11||1|2022-07-21||NovusLoading|12-23264813|
|Z20.822|1454|DIAG10|0|0|12||1|2022-07-21||NovusLoading|12-23264813|
|R07.2|1454|DIAG10|1|0|13||1|2022-07-21||NovusLoading|12-23264813|
|5A1955Z|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23264813|
|0BH17EZ|1454|PROC10|0|0|1||1|2022-07-21||NovusLoading|12-23264813|
|5A12012|1454|PROC10|0|0|2||1|2022-07-21||NovusLoading|12-23264813|
|5A09357|1454|PROC10|0|0|3||1|2022-07-21||NovusLoading|12-23264813|
|3E033XZ|1454|PROC10|0|0|4||1|2022-07-21||NovusLoading|12-23264813|
|0DH67UZ|1454|PROC10|0|0|5||1|2022-07-21||NovusLoading|12-23264813|
|3E0G76Z|1454|PROC10|0|0|6||1|2022-07-21||NovusLoading|12-23264813|
|D70.2|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23269549|
|N05.8|1454|DIAG10|0|0|1||1|2022-07-21||NovusLoading|12-23269549|
|N18.31|1454|DIAG10|0|0|2||1|2022-07-21||NovusLoading|12-23269549|
|D84.9|1454|DIAG10|0|0|3||1|2022-07-21||NovusLoading|12-23269549|
|Z94.0|1454|DIAG10|0|0|4||1|2022-07-21||NovusLoading|12-23269549|
|Z79.899|1454|DIAG10|0|0|5||1|2022-07-21||NovusLoading|12-23269549|
|T86.19|1454|DIAG10|0|0|6||1|2022-07-21||NovusLoading|12-23269549|
|N05.A|1454|DIAG10|0|0|7||1|2022-07-21||NovusLoading|12-23269549|
|10E0XZZ|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23269549|
|0UQMXZZ|1454|PROC10|0|0|1||1|2022-07-21||NovusLoading|12-23269549|
|Z00.01|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23270345|
|N18.2|1454|DIAG10|0|0|1||1|2022-07-21||NovusLoading|12-23270345|
|E11.69|1454|DIAG10|0|0|2||1|2022-07-21||NovusLoading|12-23270345|
|E78.2|1454|DIAG10|0|0|3||1|2022-07-21||NovusLoading|12-23270345|
|K86.1|1454|DIAG10|0|0|4||1|2022-07-21||NovusLoading|12-23270345|
|Z68.31|1454|DIAG10|0|0|5||1|2022-07-21||NovusLoading|12-23270345|
|Z13.89|1454|DIAG10|0|0|6||1|2022-07-21||NovusLoading|12-23270345|
|E11.22|1454|DIAG10|0|0|7||1|2022-07-21||NovusLoading|12-23270345|
|5A09457|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23270345|
|F03.91|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23270514|
|G30.9|1454|DIAG10|1|0|1||1|2022-07-21||NovusLoading|12-23270514|
|XW033E5|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23270514|
|5A1955Z|1454|PROC10|0|0|1||1|2022-07-21||NovusLoading|12-23270514|
|XW033H5|1454|PROC10|0|0|2||1|2022-07-21||NovusLoading|12-23270514|
|0BH18EZ|1454|PROC10|0|0|3||1|2022-07-21||NovusLoading|12-23270514|
|05HM33Z|1454|PROC10|0|0|4||1|2022-07-21||NovusLoading|12-23270514|
|05HN33Z|1454|PROC10|0|0|5||1|2022-07-21||NovusLoading|12-23270514|
|B544ZZA|1454|PROC10|0|0|6||1|2022-07-21||NovusLoading|12-23270514|
|S32.018A|1454|DIAG10|1|1|0||1|2022-07-21||NovusLoading|12-23270519|
|S27.2XXA|1454|DIAG10|0|0|1||1|2022-07-21||NovusLoading|12-23270519|
|S45.102A|1454|DIAG10|0|0|2||1|2022-07-21||NovusLoading|12-23270519|
|A41.9|1454|DIAG10|0|0|3||1|2022-07-21||NovusLoading|12-23270519|
|R57.1|1454|DIAG10|0|0|4||1|2022-07-21||NovusLoading|12-23270519|
|S24.104A|1454|DIAG10|0|0|5||1|2022-07-21||NovusLoading|12-23270519|
|S24.102A|1454|DIAG10|0|0|6||1|2022-07-21||NovusLoading|12-23270519|
|R40.2112|1454|DIAG10|0|0|7||1|2022-07-21||NovusLoading|12-23270519|
|R40.2212|1454|DIAG10|0|0|8||1|2022-07-21||NovusLoading|12-23270519|
|R40.2342|1454|DIAG10|0|0|9||1|2022-07-21||NovusLoading|12-23270519|
|S21.331A|1454|DIAG10|0|0|10||1|2022-07-21||NovusLoading|12-23270519|
|S21.332A|1454|DIAG10|0|0|11||1|2022-07-21||NovusLoading|12-23270519|
|J85.0|1454|DIAG10|0|0|12||1|2022-07-21||NovusLoading|12-23270519|
|I21.A1|1454|DIAG10|0|0|13||1|2022-07-21||NovusLoading|12-23270519|
|J18.9|1454|DIAG10|0|0|14||1|2022-07-21||NovusLoading|12-23270519|
|S22.32XA|1454|DIAG10|0|0|15||1|2022-07-21||NovusLoading|12-23270519|
|N17.9|1454|DIAG10|0|0|16||1|2022-07-21||NovusLoading|12-23270519|
|I82.401|1454|DIAG10|0|0|17||1|2022-07-21||NovusLoading|12-23270519|
|K59.2|1454|DIAG10|0|0|18||1|2022-07-21||NovusLoading|12-23270519|
|J98.19|1454|DIAG10|0|0|19||1|2022-07-21||NovusLoading|12-23270519|
|0D798DZ|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23270519|
|0DJ08ZZ|1454|PROC10|0|0|1||1|2022-07-21||NovusLoading|12-23270519|
|C86.4|1454|DIAG10|0|1|0||1|2022-07-21||NovusLoading|12-23270520|
|5A09357|1454|PROC10|0|1|0||1|2022-07-21||NovusLoading|12-23270520|
